### PR TITLE
chore: 🚬  refactor verification api

### PR DESCRIPTION
**Pull request by scott-dn**
## Why?

Refactor verification api.

## How?

- Refactor verification api.

## Tickets?

- N/A

## Contribution checklist?

- [x] The commit messages are detailed
- [x] It does not break existing features (unless required)
- [x] I have performed a self-review of my own code
- [x] Documentation has been updated to reflect the changes
- [x] Tests have been added or updated to reflect the changes
- [x] All code formatting pass
- [x] All lints pass
- [x] All tests pass

## Security checklist?

- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] The UI is escaping output (to prevent XSS)
- [ ] Sensitive data has been identified and is being protected properly

## Demo?

Optionally, provide any screenshot, gif or small video.

### CODE REVIEW

File: services/cover/src/service/cover.rs [hunk 1]
```diff
@@ -58,25 +58,19 @@ pub fn update_progress(update_progress: UpdateProgress) -> Result<(), Error> {
         .map_err(|e| e.into())
 }
 
-pub fn add_verification(
+pub fn submit_verification(
     caller_id: CallerId,
```
Comment (rbialek) [inline services/cover/src/service/cover.rs:62]: Please keep add and update verification apis - we might switch to them when we introduce a security layer

**Review by rbialek — CHANGES_REQUESTED** [review]
Pls, just add the submit_verification api

File: services/cover/src/service/cover.rs (hunk 1 above)
Comment (scott-dn) [reply services/cover/src/service/cover.rs:62]: so may I closed the PR @rbialek, since you don't need it anymore?

Comment (rbialek) [reply services/cover/src/service/cover.rs:62]: I need submit_verification on top of add_verifcation and update_verification

Comment (scott-dn) [reply services/cover/src/service/cover.rs:62]: it's exactly the code is doing, isn't it?

Comment (rbialek) [reply services/cover/src/service/cover.rs:62]: yes, but we loose the ability to call add update from the outside.

**Review by rbialek — APPROVED** [review]
I would prefer to keep the add/update method side-by-side with the new submit method. 
OK with me to remove them though. 
Pls merge

**Pull request closed without merging**